src/components/UserInfo.js

```diff
@@ -7,6 +7,10 @@ export default class UserInfo {
     this._bioElement = document.querySelector(this._bioSelector);
   }
 
+  setAvatarPic(element, link) {
+    element.src = link;
```

Comment:
You need to fine the avatar in line 7 inside `UserInfo` using the avatar selector which should be passed into the constructor


```js

  constructor(nameSelector, bioSelector, avatarSelector) {

```


Here will be


```js

  setAvatarPic(link) {
    this._element.src = link;
  }
```